cmd/cri-containerd/cri_containerd.go

```diff
@@ -38,6 +40,11 @@ func main() {
 	glog.V(2).Infof("Run cri-containerd grpc server on socket %q", o.CRIContainerdSocketPath)
 	service := server.NewCRIContainerdService(conn)
 	s := server.NewCRIContainerdServer(o.CRIContainerdSocketPath, service, service)
+
+	if o.ContainerdVersion {
```

Comment:
We should at the beginning. `cri-containerd --version` should not do anything else.